index.js

```diff
@@ -1,5 +1,103 @@
 // Calculator project
-const display = document.querySelector('.display');
-const one = document.querySelector('.one');
+display = document.querySelector('.display');
+let result = ''
+let completed = false
 
-const total = 0;
+let number = [];
+for (var i = 0; i <= 9; ++i) {
+    number[i] = document.querySelector(".number" + i);
+    number[i].addEventListener("click", function () { addDigit() });
+}
+
+remove = document.querySelector('.delete');
+remove.addEventListener("click", function () { removeDigit() });
+clear = document.querySelector('.clear');
+clear.addEventListener("click", function () { clearDisplay() });
+equal = document.querySelector('.equal');
+equal.addEventListener("click", function () { Calc() });
+add = document.querySelector('.add');
+add.addEventListener("click", function () { addOperator("+") });
+substract = document.querySelector('.substract');
+substract.addEventListener("click", function () { addOperator("-") })
+multiply = document.querySelector('.multiply');
+multiply.addEventListener("click", function () { addOperator("*") })
+divide = document.querySelector('.divide');
+divide.addEventListener("click", function () { addOperator("/") })
+preview = document.querySelector('.preview');
+
+function clearDisplay() {
+    display.innerHTML = 0;
+    result = ''
+    preview.innerHTML = '';
+}
+
+function addDigit() {
+    if (completed) {
+        clearDisplay()
+        completed = false
+    }
+    result = result + this.event.target.innerHTML
```

Comment:
nice that you did event delegation here. Could you elaborate on this for the others during class?